stylesheets/styles.css

```diff
@@ -19,8 +19,8 @@ body, html {
    
 .logo {
     display: flex;
-    height: 180%;
-    width: 82rem;
+    height: 120%;
+    width: 35rem;
 }
```

Comment:
Thoughts: 

- The logo seems to fit most devices, which is great! 🙂 However, it doesn't seem to fit iPhone 5/SE or smaller-screened devices. Suggestion: 
```
.logo {
    display: flex;
    height: 100%;
    width: 30rem;
}
```
- Could we possibly change the height to 100%? As it currently is at 120%, the logo is overflowing out of the 'header' container.

Reply:
I agree. Changing the height to 100% fixes the display issues on smaller devices.